assets/queries/terraform/alicloud/launch_template_is_not_encrypted/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "1455cb21-1d48-46d6-8ae3-cef911b71fd5",
+  "queryName": "Launch Template Is Not Encrypted",
+  "severity": "HIGH",
+  "category": "Encryption",
+  "descriptionText": "Data stored in the ECS Launch Template is not encrypted",
```

Comment:
Please, improve the `descriptionText` in this query and in the following ones:

- `kics\assets\queries\ansible\aws\launch_configuration_is_not_encrypted\metadata.json`
- `kics\assets\queries\terraform\aws\launch_configuration_is_not_encrypted\metadata.json`